Tabloid/client/src/components/Category/ListCategories.js

```diff
@@ -1,18 +1,29 @@
 import React, { useState, useEffect } from "react"
-import { Table } from "reactstrap"
+import { useNavigate } from "react-router-dom"
+import { Button, Table } from "reactstrap"
 import { getAllCategories } from "../../modules/categoryManager"
+import Pagination from "../Helpers/Pagination"
 import Category from "./Category"
 
 const ListCategories = () => {
-    const [categories, setCategories] = useState([])
+    const [categories, setCategories] = useState([]),
+        [offset, setOffset] = useState(0),
+        [total, setTotal] = useState(0),
+        [increment] = useState(10),
```

Comment:
you have this number hardcoded on the the client and server side. Can you figure out how to put the client in charge of this number?